controllers/district_controller.py

```diff
@@ -94,6 +100,16 @@ def _render(self, district_abbrev, year=None, explicit_year=False):
             middle_value += 1
         teams_a, teams_b = teams[:middle_value], teams[middle_value:]
 
+        # Currently Competing Team Status
+        live_events_with_teams = []
+        for event, teams_future in zip(live_events, live_eventteams_futures):
+            live_teams_in_district = TeamHelper.sortTeams(filter(lambda t: t in teams, teams_future.get_result()))
+
+            teams_and_statuses = []
+            for team in live_teams_in_district:
+                teams_and_statuses.append((team, EventTeamStatusHelper.generateTeamAtEventStatus(team.key_name, event)))
+            live_events_with_teams.append((event, teams_and_statuses))
```

Comment:
https://github.com/the-blue-alliance/the-blue-alliance/pull/1484/files#diff-5208a9f63fe1ae1498390f74bb99c2c9R18